server/channels/store/sqlstore/store.go

```diff
@@ -1002,18 +1003,57 @@ func (ss *SqlStore) DropAllTables() {
 			    FROM   pg_class
 			    WHERE  relkind = 'r'  -- only tables
 			    AND    relnamespace = 'public'::regnamespace
-				AND NOT relname = 'db_migrations'
+				AND NOT (
+                  relname = 'db_migrations' OR
+                  relname = 'focalboard_schema_migrations' OR
+                  relname = 'focalboard_boards' OR
+                  relname = 'focalboard_blocks'
+                )
 			   );
 			END
 			$func$;`)
+		tableSchemaFn = "current_schema()"
 	} else {
 		tables := []string{}
 		ss.masterX.Select(&tables, `show tables`)
 		for _, t := range tables {
-			if t != "db_migrations" {
+			if t != "db_migrations" &&
+				t != "focalboard_schema_migrations" &&
```

Comment:
Is it now safe to hard-code the table prefix?

Reply:
In the channels helper it is, as when boards starts as product it always uses that prefix